.github/workflows/conformance-clustermesh.yaml

```diff
@@ -370,7 +370,7 @@ jobs:
             envsubst < ./.github/kind-config.yaml.tmpl > ./.github/kind-config-cluster2.yaml
 
       - name: Create Kind cluster 1
-        uses: helm/kind-action@0025e74a8c7512023d06dc019c617aa3cf561fde # v1.10.0
+        uses: helm/kind-action@9fdad0686e6f19fcd572f62516f5e0436f562ee7 # v1.10.0
```

Comment:
Hmm, this looks confusing to me, because the SHA no longer matches the tag mentioned in the comment. I'm also not sure how renovate would react due to this mismatch. Should we wait for v1.11 to be released? (https://github.com/helm/kind-action/issues/128)